MuMuPiKPAT/test/sanjay/selector/psiPrimePiK_MC.C

```diff
@@ -3044,11 +3061,25 @@ Bool_t psiPrimePiK_MC::Process(Long64_t entry)
 		      hmyPsiPKPiMassCutsSelAlt->Fill( B0Mass_1B0_hardCuts );
 		      hmyPsiPPiPiMassCutsSelAlt->Fill( (PsiPp4_orig + PiPip4_orig).M() );
 
-		      if ( fabs(myB0MassAlt - B0_massFit) < B0_signal*B0_sigmaFit )
+		      if ( fabs(myB0MassAlt - B0_massFit) < B0_signal*B0_sigmaFit ) {
 			psi2SPiMassSq_vs_KPiMassSq_CutsPeak->Fill( KPip4_orig.M2(), Zp4_orig.M2() ) ;
+                        hardcuts_vars_sig[0] = KPi_hC;
+                        hardcuts_vars_sig[1] = psiPi_hC;
+                        hardcuts_vars_sig[2] = cosTheta_psi_hC;
+                        hardcuts_vars_sig[3] = phi_hC;
+                        hardcuts_vars_sig[4] = B0beauty_hC;
+                        hardcuts_variables_sig->Fill();
```

Comment:
The filling must be done here: https://github.com/lecriste/Z4430/pull/1/files#diff-2aef6870c472c96f6a5aec4107f14debR4234
Do you agree?

Reply:
I did it the same way mva_variables_sig (bkg) (all) are filled. Is that ok?

Reply:
`mva_variables_sig` is filled for events with only 1 candidate:
https://github.com/lecriste/Z4430/blob/11e052bf44da040cb1aeb2474a47a8a7fb16869e/MuMuPiKPAT/test/sanjay/selector/psiPrimePiK_MC.C#L3963

You want to require the same for `hardcuts_variables_sig`, right?